### PR TITLE
created a model for an appointment

### PR DESCRIPTION
**Pull request by garzeah**
### What this PR does (required):
- Created a model for making an appointment

### Screenshots / Videos (required):
- Not applicable

### Any information needed to test this feature (required):
- Not applicable

### CODE REVIEW

File: server/models/Appointment.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const appointmentSchema = new mongoose.Schema({
+  meeting_id: {
+    type: String,
+    required: true,
+    unique: true
+  },
```
Comment (moffatethan) [inline server/models/Appointment.js]: We should be referencing to the meeting collection, please use `mongoose.Types.ObjectId` for the property type and add a `ref: 'meeting'`

Comment (moffatethan) [reply server/models/Appointment.js]: Also, camelCase and not snake_case here please 😄

File: server/models/Appointment.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    required: true
+  },
+  time: {
+    type: Date,
+    required: true
+  },
+  timezone: {
+    type: Date,
+    required: true
+  }
```
Comment (moffatethan) [inline server/models/Appointment.js]: What is the plan with this here? Wouldn't it be better to store the users timezone as a string from a list of supported time zones in our Date library and then we can just do the conversion on the time property's date?

**Review by moffatethan — COMMENTED** [review]
Left you some comments to address. Good job!

File: server/models/Appointment.js (hunk 1 above)
Comment (garzeah) [reply server/models/Appointment.js]: Just implemented your suggestions!

File: server/models/Appointment.js (hunk 2 above)
Comment (garzeah) [reply server/models/Appointment.js]: That's a good point, I just pushed a commit that converts the type to a string. Thanks for the feedback!

**Review by moffatethan — APPROVED** [review]
Approved! The `timezone` property can be further discussed but I think right now this is a good start. Otherwise, I would take a look at the Date object as well and see if anything stands out we could utilize here.

**Pull request merged**